tests/integration/test_storage_kafka_sasl/secrets/kafka_server_jaas.conf

```diff
@@ -0,0 +1,6 @@
+KafkaServer {
+    org.apache.kafka.common.security.plain.PlainLoginModule required
+    username="admin"
+    password="admin-password"
+    user_admin="admin-password";
```

Comment:
Could you please add these usernames/passwords to `tests/integration/helpers/config_cluster.py`? Using a more specific password (e.g. similar to what is in the file: `ClickHouse_Kafka_SASL_P@ssw0rd`) might help to catch the issue of possible logging the password in tests once https://github.com/ClickHouse/ClickHouse/pull/78094 is merged.

Reply:
done